documentation/src/docs/recovery-procedures/lts-recovery-steps.md

```diff
@@ -0,0 +1,83 @@
+<!--
+Copyright Pravega Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# LTS Recovery
+The following procedure describes offline method of doing the LTS recovery. The offline method of doing the LTS recovery requires the admin to manually copy metadata chunk files to some directory.
+
+The offline method currently only supports filesystem storage type.
+
+The major inspiration for this recovery procedure has been [#7011](https://github.com/pravega/pravega/issues/7011). The way this recovery
+procedure works is by trying to restore the metadata state(state of all Segments in Pravega)
+stored in Tier-2 Segment Chunks, by parsing these raw Chunk files. This recovery procedure 
+assumes full Tier-1 loss, and as such the Tier-1 tail data or the latest updates to the system as lost.
+The aim of this recovery procedure is to get Pravega functional again after performing the recovery procedure,
+to allow clients to resume their operations. Some cases where the recovery procedure
+might be useful are ZooKeeper becoming dysfunctional or Bookkeeper disk corruptions etc, where we 
+intend to start Pravega off the data that exists in Tier-2.
+The following section describes how to perform the recovery steps.
+
+## Prerequisites
+Having pravega installed with data generated on it. 
+
+## Steps
+* ## Executing the ltsrecovery.sh script
+  ```
+  ./ltsrecovery --install
+  ```
+  This script will create a recovery pod and has the LTS directory mounted inside it, and it will uninstall **pravega**, **pravega-operator**, **bookkeeper**, **bookkeeper-operator**, **zookeeper** and **zookeeper-operator**.
+
+  Before proceeding further, make sure the script executed successfully and uninstalled all the services mentioned above.
+
+  The ltsrecovery script will only work where helm is used to install pravega.
```

Comment:
Please, use formatting on `ltsrecovery` and `helm`. Pravega starts with capital P.

Reply:
fixed.